packages/control-group/src/index.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import {ButtonProps, ButtonAppearanceType} from '@smashing/button'
+import {Radio, RadioAppearanceType} from '@smashing/radio'
+import {Checkbox, CheckboxAppearanceType} from '@smashing/checkbox'
+import {
+  ControlProps,
+  ControlGroupProps,
+  ControlAppearanceType,
+  ControlGroupAppearanceType
+} from './types'
+import {useDefaults} from '@smashing/theme'
+import {StyledButton, ControlGroupWrapper} from './styles'
+
+const ControlGroupFC: React.FC<ControlGroupProps & ButtonProps> = props => {
```

Comment:
```suggestion
const ControlGroupFC: React.FC<ControlGroupProps> = props => {
```

Are those ButtonProps required?

Reply:
fixed